opsdroid/connector/__init__.py

```diff
@@ -92,24 +165,49 @@ def configuration(self):
             bool: True for message successfully sent. False otherwise.
 
         """
-        _LOGGER.debug(_("%s connector can't react to messages"), self.name)
-        return False
+        warnings.warn(
+            "Connector.react is deprecated. Use "
+            "Connector.respond(events.Reaction(emoji)) instead.",
```

Comment:
Is `Connector.respond()` not also deprecated, as per the [warning above](https://github.com/opsdroid/opsdroid/pull/826/files#diff-cf8aeca0533e0e524ca35bb33398ad42R134)?